config/app.php

```diff
@@ -15,7 +67,7 @@
     |
     */
 
-    'preflight' => false,
+    'preflight' => env('WP_ENV', 'production') !== 'production',
```

Comment:
We don't want to assume that only the production environment shouldn't be running preflight. It goes against environment parity. Instead we assume all environments are equivalent with production by default. But even if we were to keep this, it has the same issue as before with using `env()`.

Reply:
Let's just stick to `false`.